File: PHP/Cap07/c07ex02.php
<?php
// c07ex02.php

printf("Entre um valor para a variavel <A>: ");
$A = intval(trim(fgets(STDOUT)));
printf("Entre um valor para a variavel <B>: ");
$B = intval(trim(fgets(STDOUT)));

$X = $A;
$A = $B;
$B = $X;

printf("\n");
printf("Variavel <A> com valor %d\n", $A);
printf("Variavel <B> com valor %d\n", $B);

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDOUT));

?>

File: GO/Cap07/c07ex02.go
// c07ex02.go

package main
import "fmt"

func main() {
  var A, B, X int

  fmt.Print("Entre um valor para a variavel <A>: ")
  fmt.Scan(&A)

  fmt.Print("Entre um valor para a variavel <B>: ")
  fmt.Scan(&B)

  X = A
  A = B
  B = X

  fmt.Println()
  fmt.Println("Variavel <A> com valor", A)
  fmt.Println("Variavel <B> com valor", B)

  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}
